compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -265,10 +272,22 @@ xExtensionsSet =
   , DamlSyntax
   ]
 
+-- | Extensions which we support with data-dependencies.
+dataDependableExtensions :: ES.EnumSet Extension
```

Comment:
The `Ord` instance for `Extension` has been added to GHC after we forked off. Thus, I use `EnumSet` instead of a regular `Data.Set` here.

Reply:
Seems reasonable although you could also just throw an orphan instance somewhere.